CS50p-Libraries/name.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+# Check for errors
+if len(sys.argv) < 2:
+    # print("Too few arguments")
+    sys.exit("Too few arguments")
+# elif len(sys.argv) > 2:
+#     # print('Too many arguments')
+#     sys.exit("Too many arguments")
+# else:
+# Print name tags
+
+
+# slices
+for arg in sys.argv[1:]:
+    print("Hello", "My name is", arg)
```

Comment:
[explore]: print vs logger and their use case! It would be nice to use logger instead of print bc of many other functionalities: [reference](https://stackoverflow.com/questions/6918493/in-python-why-use-logging-instead-of-print)